velox/docs/functions/spark/math.rst

```diff
@@ -66,6 +66,26 @@ Mathematical Functions
     Returns ``x`` rounded up to the nearest integer.  
     Supported types are: BIGINT and DOUBLE.
 
+.. function:: check_add(x, y) -> [same as x]
```

Comment:
nit: maybe use checked_add as the function name to correspond with the struct name CheckedAddFunction. Ditto for the others.

Reply:
Updated, thanks